src/lib/components/Communities/CommunityHeader.js

```diff
@@ -83,17 +20,48 @@ export class CommunityHeaderComponent extends React.Component {
   }
 
   render() {
-    const { communityRedux, imagePlaceholderLink } = this.props;
+    const { communityRedux, imagePlaceholderLink, setCommunity } = this.props;
```

Comment:
Can you rename `communityRedux` back to `community`?

Reply:
`community` already exists as a prop, which is passed down by jinja, and `communityRedux` is the redux state of the community

Reply:
The `community` prop passed from Jinja would be initialized in redux. I had done that in my [draft pr](https://github.com/inveniosoftware/react-invenio-deposit/pull/419/files#diff-fb22c95a502cae93ba1418946e8c3ecf35676cde6f5fd8c665df3c9ec7853d9eR61) for requesting review. Can you merge the community with the communityRedux property?